app/labs/edit/route.js

```diff
@@ -11,13 +11,22 @@ export default AbstractEditRoute.extend(AddToPatientRoute, ChargeRoute, PatientL
   modelName: 'lab',
   newTitle: t('labs.newTitle'),
   pricingCategory: 'Lab',
+  customForms: Ember.inject.service(),
 
   actions: {
     returnToAllItems() {
       this.controller.send('returnToAllItems');
     }
   },
 
+  getCustomFormData() {
```

Comment:
Looks like this code isn't used, but you need this code for forms that are set to "Always Include".  Custom forms that are set to this get added automatically on new items.